conftest.py

```diff
@@ -133,16 +143,16 @@ def small_client(small_cluster, s3_cluster_dump_url, s3_storage_options, request
 def s3_storage_options():
     return {
         "config_kwargs": {"region_name": S3_REGION},
-        "key": os.environ["AWS_ACCESS_KEY_ID"],
-        "secret": os.environ["AWS_SECRET_ACCESS_KEY"],
+        # "key": os.environ["AWS_ACCESS_KEY_ID"],
```

Comment:
Don't we need to uncomment these lines to be able to test things with CI?

Reply:
Whoop -- left over from local debugging